apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -71,6 +72,41 @@ const CurriculumCatalog = ({
     setExpandedCardKey(expandedCardKey === key ? null : key);
   };
 
+  // Get the top recommended similar curriculum based on the curriculum with the given
+  // curriculumKey
+  const getRecommendedSimilarCurriculum = curriculumKey => {
+    let recommendableCurricula = [...curriculaData];
+
+    // Get current curricula the others are being compared against and filter it out of the
+    // recommendable curriculum
+    const currCurriculumIndex = recommendableCurricula.findIndex(
+      e => e.key === curriculumKey
+    );
+    const currCurriculum = recommendableCurricula.splice(
+      currCurriculumIndex,
+      1
+    )[0];
+
+    // Filter out curricula that don't support any of the same grade levels
+    const currCurriculumGrades = currCurriculum.grade_levels;
+    recommendableCurricula = recommendableCurricula.filter(curr =>
+      curr.grade_levels
+        ?.split(',')
+        ?.some(grade => currCurriculumGrades.includes(grade))
+    );
+
+    // Get top recommended similar curriculum
+    const recommendations = getSimilarRecommendations(
+      recommendableCurricula,
+      currCurriculum.duration,
+      currCurriculum.marketing_initiative,
+      currCurriculum.school_subject,
+      currCurriculum.cs_topic
+    );
+
+    return recommendations[0];
```

Comment:
The Figma shows 2 curricula because one will be the recommended "Similar" curriculum and one will be the recommended "Stretch" curriculum. So we should only show 1 for now!

Reply:
ooo thanks!